### PR TITLE
Fix ebpf install docs

### PR DESCRIPTION
**Pull request by sridhartigera**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise/operations/ebpf/install.mdx [hunk 1]
```diff
@@ -247,6 +247,21 @@ In this example, you would use `60F939227672BC3D5A1B3EC9744B2B21.gr7.us-west-2.e
...
+
+Follow the steps in the main install guide for your platform to install the Tigera Operator (and possibly the
+Prometheus Operator). For a KOPS cluster, patch the tigera-operator deployment with dns config, so that the operator
+can resolve the apiserver DNS.
+
+```bash
+kubectl patch deployment -n tigera-operator tigera-operator -p '{"spec":{"template":{"spec":{"dnsConfig":{"nameservers":["127.0.0.53"]}}}}}'
+```
+
+However, **stop** before applying the `custom-resources.yaml`; we'll customise that file
```
Comment (bmckercher123) [inline calico-enterprise/operations/ebpf/install.mdx]: I'm afraid folks might read the first sentence and miss the note about not applying the yaml. How about this:

Follow the steps in the main install for your platform that installs the Tigera Operator, **without applying the `custom-resources.yaml`** (you will update this file in a later step in this doc).  If you have a KOPS cluster, you must also patch the tigera-operator deployment with DNS config so the operator can resolve the apiserver DNS.

**Review by bmckercher123 — CHANGES_REQUESTED** [review]
Suggest for clarity.

File: calico-enterprise/operations/ebpf/install.mdx [hunk 2]
```diff
@@ -247,6 +247,16 @@ In this example, you would use `60F939227672BC3D5A1B3EC9744B2B21.gr7.us-west-2.e
 </TabItem>
 </Tabs>
 
+### Install the Tigera Operator
+
+Follow the steps in the main install for your platform that installs the Tigera Operator, without applying the custom-resources.yaml (you will update this file in a later step in this doc). If you have a KOPS cluster, you must also patch the tigera-operator deployment with DNS config so the operator can resolve the apiserver DNS.
+
+```bash
+kubectl patch deployment -n tigera-operator tigera-operator -p '{"spec":{"template":{"spec":{"dnsConfig":{"nameservers":["127.0.0.53"]}}}}}'
+```
```
Comment (tomastigera) [inline calico-enterprise/operations/ebpf/install.mdx:258]: Shouldn't this be in some (sub)section  that desribes individual environments? Do we need that without ipv6?

Comment (sridhartigera) [reply calico-enterprise/operations/ebpf/install.mdx:258]: This is needed even for ipv4. This is applicable only for kops and no other platform. Hence I feel having sub sections is not needed. WDYT?

**Review by tomastigera — APPROVED** [review]

**Review by bmckercher123 — APPROVED** [review]
LGTM

**Pull request merged**